libres/lib/enkf/local_config.cpp

```diff
@@ -247,3 +229,33 @@ local_config_get_updatestep(const local_config_type *local_config) {
 
     return updatestep;
 }
+
+namespace {
+
+LocalObsData &get_obsdata(py::handle obj, const std::string &obs_key) {
+    auto *local_config = reinterpret_cast<local_config_type *>(
+        PyLong_AsVoidPtr(obj.attr("_BaseCClass__c_pointer").ptr()));
+    auto *obsdata = local_config_get_obsdata(local_config, obs_key.c_str());
+    return *obsdata;
+}
+
+LocalObsData copy_obsdata(py::handle obj, const std::string &obs_key,
+                          const std::string &target_key) {
+    auto *local_config = reinterpret_cast<local_config_type *>(
+        PyLong_AsVoidPtr(obj.attr("_BaseCClass__c_pointer").ptr()));
+    auto *obsdata = local_config_get_obsdata(local_config, obs_key.c_str());
+    LocalObsData copy(target_key);
+    for (const auto &node : *obsdata)
+        copy.add_node(node);
+    return copy;
+}
+
+} // namespace
+
+RES_LIB_SUBMODULE("local.local_config", m) {
+    using namespace py::literals;
+    m.def("get_obsdata_ref", &get_obsdata, "self"_a, "key"_a,
+          py::return_value_policy::reference);
+    m.def("get_obsdata_copy", &copy_obsdata, py::arg("self"), py::arg("key1"),
+          py::arg("key2"));
```

Comment:
```suggestion
    m.def("get_obsdata_copy", &copy_obsdata, "self"_a, "key1"_a,
          "key2"_a);
```

Reply:
Fixed in https://github.com/equinor/ert/pull/2966.